exporter/importables.go

```diff
@@ -527,9 +527,15 @@ var resourcesMap map[string]importable = map[string]importable{
 					log.Printf("[INFO] Group %s doesn't match %s filter", g.DisplayName, ic.match)
 					continue
 				}
+				hclblockMode := "resource"
+				genDataBlock := map[string]bool{"admins": true, "users": true}
```

Comment:
technically, `map[string]interface{}` is better because it could be used as a `set`, so we can use it as:

```
if _, exists := genDataBlock[g.DisplayName]; exists {
...
}
```